src/main/resources/application.properties

```diff
@@ -1,8 +1,5 @@
 bot.name=RandomSimulatorBot
 
-# period of new atom creation, in milliseconds
-atomCreatorBot.period=10000
```

Comment:
please lets not remove this from here and inject the value directly in the bot.xml, i feel like this is an attribute that can stay in the specific application.properties config

Reply:
I agree - that's an older commit. Not present in the PR.